src/tests/ftest/util/dmg_utils.py

```diff
@@ -338,13 +340,13 @@ def storage_led_check(self, ids=None):
         return self._get_json_result(
             ("storage", "led", "check"), ids=ids)
 
-    def storage_replace_nvme(self, old_uuid, new_uuid, no_reint=False):
+    def storage_replace_nvme(self, host, old_uuid, new_uuid):
```

Comment:
Also need to update this usage
https://github.com/daos-stack/daos/blob/f9b3f46a3d0e2b6c7555d9e4897d0eba398feca7/src/tests/ftest/vmd/led.py#L133

Reply:
good catch, thanks. fixed